src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -1612,7 +1613,12 @@ public synchronized boolean saveTown(Town town) {
         return false;
     }
 
-    @Override
+	@Override
+	public boolean savePlotGroup(PlotGroup group) {
+		return false;
```

Comment:
All of the changes to the SQLSource so far either return false or do nothing (as is the case in deleteGroup)